reframe/frontend/cli.py

```diff
@@ -1037,6 +1037,20 @@ def restrict_logging():
             )
             sys.exit(0)
 
+    if site_config.get('general/0/remote_detect') and \
+            site_config.get('general/0/remote_command'):
+        remote_commands = site_config.get('general/0/remote_command')
+        detect_topology_cmd = "reframe --detect-host-topology=topo.json"
+        detect_topology_found = [
+            s for s in remote_commands if detect_topology_cmd in s
+        ]
+        if not detect_topology_found:
+            remote_cmd_args = site_config.get('general/0/remote_command')
+            printer.debug(
+                "Adding ReFrame topology detection to the curstom command"
+            )
+            remote_cmd_args.append(detect_topology_cmd)
```

Comment:
This is not needed. We should always add the reframe command to execute.

Reply:
What if the command needs to be `./bin/reframe ...`, like it is the case for the bootstrap option?

Reply:
I guess that would be the case when sb wants to clone explicitly and bootstrap it, because by default we make a copy (NB: the method we discuss is something extra to what we do now). Well, in this case the user could set the `PATH` to the local `bin/`:

```python
remote_install = ['git clone ... && cd reframe && ./boostrap.sh && cd -',
                  'export PATH=$PWD/reframe/bin:$PATH']
```